chalice/chalicelib/utilities.py

```diff
@@ -29,15 +29,17 @@ def from_json(self, json_data):
         data = json.loads(json_data)
         return data
 
-    def read_file(self, path):
-        with open(path, 'r') as file:
+    def read_file(self, path, binary=False):
+        mode = 'rb' if binary else 'r'
```

Comment:
@danjoneslf there are two variables called 'mode' in this commit, one is this file handling and the other is the cli mode var. Different files to be fair but maybe we should change one?

Reply:
Hmm. They're separate variables because they're in different scopes / contexts but I take your point. @0atman - do you have an opinion?